packages/edit-post/src/components/layout/index.js

```diff
@@ -220,11 +220,17 @@ function Layout() {
 		[ entitiesSavedStatesCallback ]
 	);
 
+	// We need to add the show-icon-labels class to the body element so it is applied to modals.
+	if ( showIconLabels ) {
+		document.body.classList.add( 'show-icon-labels' );
+	} else {
+		document.body.classList.remove( 'show-icon-labels' );
+	}
```

Comment:
Since it's a side effect, should we put this in an `useEffect`?

Reply:
yeh, have moved to a useEffect